### PR TITLE
Extend installation dialog

### PR DESCRIPTION
**Pull request by joseivanlopez**
Needed for PBI: https://trello.com/c/53IEUxrB/577-susecaas-platform-4-p2-1099485-caasp-specific-partitioning-warning-dialog-missing-in-storage-ng

### CODE REVIEW

File: library/general/src/lib/ui/installation_dialog.rb [hunk 1]
```diff
@@ -135,12 +135,26 @@ def create_dialog
...
         help_text,
-        Yast::GetInstArgs.enable_back,
-        Yast::GetInstArgs.enable_next || Yast::Mode.normal
+        enable_back,
+        enable_next
       )
       true
     end
 
+    # Whether the back button is enabled
```
Comment (jreidinger) [inline library/general/src/lib/ui/installation_dialog.rb]: please document behavior in this implementation ( so read options from client arguments ).

File: library/wizard/src/modules/Wizard.rb [hunk 2]
```diff
@@ -92,6 +92,42 @@ def haveFancyUI
...
+        HWeight(
+          1,
+          ReplacePoint(
+            Id(:rep_next),
+            PushButton(Id(:next), Opt(:key_F10, :default), Label.NextButton)
+          )
+        )
+      )
+    end
+
```
Comment (jreidinger) [inline library/wizard/src/modules/Wizard.rb]: hmm, I am not much happy with this additional code increase which result that this ruby file has almost 2k lines. I know that it is a bit more work, but is possible to somehow refactor ( in sense of API and functionality) this module to reduce huge duplication here and decrease size?

Comment (joseivanlopez) [reply library/wizard/src/modules/Wizard.rb]: Yes, maybe we can refac that code, but the problem I see is that thd changes here are for SP0, so I prefer a more conservative change here. I would prefer to do such refactorization as part of another PBI, and not for SP0.

**Comment by joseivanlopez** [comment]
Finally these changes were not needed, see https://github.com/yast/yast-storage-ng/pull/821.

**Pull request closed without merging**